src/main/java/edu/greenblitz/robotName/commands/shooter/pivot/PivotDefaultCommand.java

```diff
@@ -3,16 +3,12 @@
 import edu.greenblitz.robotName.Robot;
 
 public class PivotDefaultCommand extends PivotCommand{
-	
-	@Override
-	public void initialize() {
-		pivot.standInPlace();
-	}
-	
+
 	@Override
 	public void execute() {
```

Comment:
What will happen not in simulation?

Reply:
At the end of the elbow, wrist, pivot I do the stand in place because of sequence commands that require even when the arm or pivot not moving

Reply:
Hevanti. So this is It’s a little out of convention… why not add a default command for all mechanisms to enforce the standInPlace (or the similar function), but not critical